Composer/packages/adaptive-form/src/components/fields/StringField.tsx

```diff
@@ -40,8 +40,17 @@ export const StringField: React.FC<FieldProps<string>> = function StringField(pr
     error,
     uiOptions,
     required,
+    focused,
   } = props;
 
+  const textFieldRef = React.createRef<ITextField>();
+
+  useEffect(() => {
+    if (focused && textFieldRef.current) {
```

Comment:
Do you also need to run this when the control is mounted?

Reply:
Nop, we don't want fields auto focused when they appear on the screen.